### ISSUE TITLE
Dockerfile missing

### ISSUE THREAD

**Issue by markeijsermans** [JFrogDev/artifactory-docker-examples#43]
The Dockerfile & source for the images are missing:
* `docker.bintray.io/jfrog/nginx-artifactory-pro:5.5.1`
* `docker.bintray.io/jfrog/artifactory-pro:5.5.1`

There's quite a lot going on with the entrypoints, and fixing issues/forking is difficult.

Diff'ing the history, It seems the postgres image is an unmodifed `postgres:9.5.2` image.

**Comment by eldada** [comment]
@markeijsermans - we have plans on adding the files. No time estimate yet - sorry.

**Comment by eldada** [comment]
Closing for now. The files are in process of being added.

**Comment by MrShark** [comment]
Unfortunate close, since it's not solved yet and the closing makes it harder to find. (Read as: I first looked att open issues, then searched for it a couple of hours and then found this. Making me trust JFrog less.)

Is there any time estimate? Even one on the level: days, months or years would be useful to decide if I should do the reverse engineering to be able to deploy.

**Comment by eldada** [comment]
@MrShark - the addition of the Dockerfiles for the Artifactory Docker images is in one of the coming minor releases (weeks away).

**Comment by eldada** [comment]
@markeijsermans, @MrShark,
The newly released Artifactory 5.7.0 now includes the Dockerfile(s) for the relevant images.
You can see the content with a simple `docker run` command:
```
# OpenJDK base image Dockerfile
$ docker run --rm -t --entrypoint="/bin/sh" docker.bintray.io/jfrog/artifactory-pro:5.7.0 -c "cat /docker/openjdk/Dockerfile"

# Artifactory Dockerfile
$ docker run --rm -t --entrypoint="/bin/sh" docker.bintray.io/jfrog/artifactory-pro:5.7.0 -c "cat /docker/artifactory-pro/Dockerfile.artifactory"

# Artifactory Nginx Dockerfile
$ docker run --rm -t --entrypoint="/bin/sh" docker.bintray.io/jfrog/nginx-artifactory-pro:5.7.0 -c "cat /docker/nginx-artifactory-pro/Dockerfile.nginx"
```

I hope this meets your needs.
